<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DATEditorClass</class>
 <widget class="QMainWindow" name="DATEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>892</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DAT Editor - DAT文件编辑器</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTableView" name="tableView">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>872</width>
     <height>26</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="separator"/>
    <addaction name="action_Import"/>
    <addaction name="action_Export"/>
    <addaction name="separator"/>
    <addaction name="action_Close"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="action_MulLine"/>
    <addaction name="separator"/>
    <addaction name="action_Find"/>
    <addaction name="action_FindNext"/>
    <addaction name="action_Replace"/>
    <addaction name="action_ReplaceNext"/>
    <addaction name="action_ReplaceAll"/>
    <addaction name="separator"/>
    <addaction name="action_Row"/>
    <addaction name="action_Column"/>
   </widget>
   <widget class="QMenu" name="menu_About">
    <property name="title">
     <string>关于(&amp;A)</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_About"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Open"/>
   <addaction name="action_Import"/>
   <addaction name="action_Export"/>
   <addaction name="action_Close"/>
   <addaction name="action_Find"/>
   <addaction name="action_Replace"/>
   <addaction name="action_Row"/>
   <addaction name="action_Column"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Open">
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_SaveAs">
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>关闭(&amp;C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>退出(&amp;Q)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_MulLine">
   <property name="text">
    <string>多行编辑(&amp;M)</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="action_Find">
   <property name="text">
    <string>查找(&amp;F)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_FindNext">
   <property name="text">
    <string>查找下一个(&amp;N)</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="action_Row">
   <property name="text">
    <string>跳转到行(&amp;R)</string>
   </property>
  </action>
  <action name="action_Column">
   <property name="text">
    <string>跳转到列(&amp;C)</string>
   </property>
  </action>
  <action name="action_Replace">
   <property name="text">
    <string>替换(&amp;R)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="action_ReplaceAll">
   <property name="text">
    <string>替换全部(&amp;A)</string>
   </property>
  </action>
  <action name="action_ReplaceNext">
   <property name="text">
    <string>替换下一个(&amp;E)</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>关于(&amp;A)</string>
   </property>
  </action>
  <action name="action_Import">
   <property name="text">
    <string>导入(&amp;I)</string>
   </property>
  </action>
  <action name="action_Export">
   <property name="text">
    <string>导出(&amp;E)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="DATEditor.qrc"/>
 </resources>
 <connections/>
</ui>
